e2e/indexer.e2e-spec.ts

```diff
@@ -21,46 +21,64 @@ describe('Indexer', () => {
         await shutdownDep();
     });
 
-    it('p2wpkh - should ensure that the correct silent block is fetched', async () => {
-        const taprootOutput = walletHelper.generateAddresses(1, 'p2tr')[0];
-        const p2wkhOutputs = walletHelper.generateAddresses(6, 'p2wpkh');
-        const utxos: UTXO[] = [];
+    const addressTypes: AddressType[] = [
+        AddressType.P2WPKH,
+        AddressType.P2TR,
+        AddressType.P2PKH,
+        AddressType.P2SH_P2WPKH,
+    ];
 
-        for (const output of p2wkhOutputs) {
-            const utxo = await walletHelper.addFundToUTXO(output, 1);
-            utxos.push(utxo);
-        }
+    it.each(addressTypes)(
```

Comment:
This should achieve the same result
```suggestion
    it.each(Object.values<AddressType>(AddressType))(
```